packages/theme/css/characteristics/container.css

```diff
@@ -15,4 +15,7 @@
   --salt-container-tertiary-background-disabled: var(--salt-palette-neutral-tertiary-background-disabled);
   --salt-container-tertiary-borderColor: var(--salt-palette-neutral-border);
   --salt-container-tertiary-borderColor-disabled: var(--salt-palette-neutral-border-disabled);
+
+  --salt-container-alpha-background: var(--salt-palette-alpha-low);
```

Comment:
Based on discussions today I think we can go ahead and make this:
--salt-container-ghost-background
--salt-container-ghost-border